docs/faq.rst

```diff
@@ -3,3 +3,154 @@
 ====================================
 Frequently Asked Questions
 ====================================
+
+.. _github issues: https://github.com/refnx/refellips/issues
+.. _refractiveindex.info: https://refractiveindex.info
+.. _Markel: https://doi.org/10.1364/JOSAA.33.001244
+.. _Humlicek: https://doi.org/10.1007/978-3-642-33956-1_3
+.. _getting started: getting_started.ipynb#Saving-the-objective
+
+A list of common questions.
+
+
+What's the best way to ask for help or submit a bug report?
+-----------------------------------------------------------
+
+If you have any questions on the use of *refellips* or calculations
+performed by *refellips* please `contact us <mailto:andyfaff+refellips@gmail.com>`_.
+If you find a bug in the code or documentation, please use
+`GitHub Issues`_.
+
+
+What are the 'fronting' and 'backing' media?
+--------------------------------------------
+
+The 'fronting' and 'backing' media are infinite. The 'fronting' medium carries
+the incident beam of radiation, whilst the 'backing' medium will carry the
+transmitted beam away from the interface. In short, the fronting media
+is the medium that the radiation interacts with first, and the backing
+media is the medium in which the radiation interacts with last.
+
+For example, consider a system with an oxidised silicon wafer where the
+ambient material is air. The fronting media would be air and the backing
+media would be silicon.
+
+
+What formats/types of ellipsometry data does *refellips* handle?
+----------------------------------------------------------------
+
+*refellips* has the capability of loading data directly from both Accurion EP3
+and EP4 ellipsometers, as well Horiba ellipsometers using `open_EP4file`
+and `open_HORIBAfile` respectively.
+
+Alternatively, users also have the option to load in other datasets using
+`DataSE`. Files loaded using `DataSE` must contain four columns (with header):
+wavelength, angle of incidence, psi and delta.
+
+
+Where do I find dispersion curves for a material?
+-------------------------------------------------
+
+*refellips* contains preloaded dispersion curves for select materials, which
+are accessible by the `load_material` function. These materials were sourced
+from `refractiveindex.info`_, and include air, a void, water, dimethyl sulfoxide,
+silicon, silica, gold, aluminium oxide, polystyrene, poly(N-isopropylacrylamide)
+(PNIPAM) and a material which represents a diffuse polymer.
+
+If required, users can download their own dispersion curves from
+`refractiveindex.info`_ and load them into *refellips* using::
+
+    my_material = RI("my_dispersion.csv")
+
+The loaded file must contain at least two columns, assumed to be wavelength
+(in microns) and refractive index. If three columns are provided the third is
+loaded as the extinction coefficient.
+
+Alternatively, users have the option to specify Cauchy parameters (a,b,c) for their material::
```

Comment:
Can we have line length 80 in the file

Reply:
Not sure I follow

Reply:
We have a max line length of 80 characters in all the Python files. Could we have the same in RST please? (except equation blocks, etc)